rubin_sim/scheduler/utils/sky_area.py

```diff
@@ -542,7 +541,9 @@ def combine_maps(self, trim_overlap=True):
             # 'self.dustfree' is already 0/1 dust-acceptable or not
             with warnings.catch_warnings():
                 warnings.simplefilter('ignore', category=UserWarning)
+                # XXX--magic number
                 dustfree = hp.smoothing(self.dustfree, fwhm=np.radians(3))
+            # XXX--magic numbers
```

Comment:
```#XXX --magic numbers``` could be replaced with

```
# Choosing whether to assign a particular healpix to the dust-free WFD vs. the galactic WFD 
# (which have different filter balances), is based on the underlying dust-map values. 
# The choice of 0.3 means that healpixels are more likely to be assigned to dust-free WFD than galactic plane,
# but that if a pixel corresponds to a very dusty region, it will be assigned to galactic plane. 
# These numbers based generally on trial-and-error.